src/quicktype-core/ConvenienceRenderer.ts

```diff
@@ -644,30 +644,34 @@ export abstract class ConvenienceRenderer extends Renderer {
         this._alphabetizeProperties = value;
     }
 
+    protected getAlphabetizeProperties(): boolean {
+        return this._alphabetizeProperties;
+    }
+
     // Returns the number of properties defined for the specified object type.
     protected propertyCount(o: ObjectType): number {
         const propertyNames = defined(this._propertyNamesStoreView).get(o);
         return propertyNames.size;
     }
 
+    protected sortClassProperties(_p: ClassProperty, jsonName: string): number | string {
```

Comment:
Please make this function actually sort the properties, and not just return a key.  First its name implies that it sorts, second `Array.sort` is not guaranteed to be stable in JS, so returning `0` doesn't necessarily mean the properties will be left in their original order.

Reply:
done